os-datavirt/added/launch/teiid.sh

```diff
@@ -16,6 +16,9 @@ function prepareEnv() {
   unset DATAVIRT_USERS
   unset DATAVIRT_USER_PASSWORDS
   unset DATAVIRT_USER_GROUPS
+  unset JDBC_SECURITY_DOMAIN
+  unset ODBC_SECURITY_DOMAIN
+  unset ODATA_SECURITY_DOMAIN
```

Comment:
I think this discussion falls on generalizing transports configuration, where the user can choose what transports to use.